dubbo-build-tools/pom.xml

```diff
@@ -84,6 +84,13 @@
         </plugins>
       </build>
     </profile>
+
+    <profile>
+      <id>turn-off-spotless</id>
```

Comment:
How about rename spotless profile to `<id>spotless</id>` and disable it with `-P-spotless`

Reply:
I can't just rename the profile to 'spotless' since the Spotless Plugin is enabled by default. However, for a better understanding, I inverted the default action and re-introduce a profile called 'spotless' to enable the plugin.